web/test/src/tests/dandisetsPage.test.js

```diff
@@ -26,4 +27,60 @@ describe('dandisets page', () => {
     await expect(page).toMatch(`Contact ${lastName}, ${firstName}`);
     await expect(page).toMatch(`Updated on ${moment(new Date()).format('LL')}`);
   });
+
+  it('search for dandisets', async () => {
+    await registerNewUser();
+
+    const dandisetIdentifierNameMapping = new Map();
+
+    // Create 10 dandisets
+    for (let i = 0; i < 10; i += 1) {
+      // Generate random dandiset name and description
+      const name = faker.lorem.words();
+      const description = faker.lorem.sentences();
+
+      /* eslint-disable no-await-in-loop */
```

Comment:
It seems like this sort of rule should be disabled in the entire tests folder, right?

Reply:
This is the only place currently where we use `await` inside a loop, but yeah I guess it would be good to include as a global setting for the tests directory since this is one of the exceptions where awaiting inside a loop makes sense.